CONTRIBUTING.md

```diff
@@ -167,6 +167,20 @@ sbt
 akka-cluster/scalafmtAll
 akka-persistence/scalafmtAll
 ```
+And you can format the Scala source code with keeping the *import*s sorted with:
```

Comment:
is this comment correct? Looks like it sorts the imports without formatting the Scala source?

Reply:
@hepin1989 ping?